storybook/storybook-docs/src/grid.stories.mdx

```diff
@@ -4,110 +4,153 @@ import { Meta } from "@storybook/blocks";
 Hier worden geen elementen over de marges heen geplaatst.
 
-### Niet onbeperkt breed
+### Menubalk
 
-Een maximale breedte voor het grid zorgt ervoor dat de elementen van een pagina niet al te ver van elkaar verwijderd raken.
-Ook wordt de witruimte en typografie dan niet extreem groot.
+Applicaties hebben in het brede grid een verticale menubalk aan de linkerzijde.
+Deze staat naast het grid, niet erin, en heeft een vaste breedte van 112 pixels.
 
-Voor websites is de maximale breedte van het grid 1432 pixels.
-In vensters van minimaal 1600 pixels breed heeft het grid deze breedte – de marges bedragen hier 84 pixels.
+In het middelbrede en smalle grid wordt het menu verticaal.
+In deze context neemt het grid wel de hele breedte van het venster in.
+
+### Maximale breedte
 
-De maximale breedte kan ook worden ingesteld op 1896 pixels.
-Inclusief de marges van 108 pixels komt de totale breedte hier op 2112 pixels.
+Een maximale breedte voor het grid zorgt ervoor dat witruimte en typografie niet onbeperkt doorgroeien.
+Zo raken de elementen van een pagina niet te ver van elkaar verwijderd en blijft tekst prettig leesbaar.
 
-Voor applicaties is er geen maximale breedte.
+Voor websites, in het ruimtelijke thema, is de maximale breedte van het grid 1432 pixels.
+In vensters van minimaal 1600 pixels breed heeft het grid deze breedte – de marges bedragen hier 84 pixels.
 
-Er is geen minimale breedte.
+Applicaties, die het compacte thema gebruiken, nemen de volledige breedte van het venster in.
+
+Er is geen expliciete minimale breedte.
 Zelfs in vensters smaller dan 320 pixels, voor zover die in de praktijk voorkomen,
 blijft het grid zich schalen naar de beschikbare breedte.
 Voor het gemak speelt de breedte van 320 wel een rol in de documentatie.
 
-### De maatvoeringen per variant
+## Maatvoering
+
+Deze ontwerpkeuzes hebben in de praktijk het volgende resultaat.
 
-Op het startpunt van de drie varianten van het grid hebben de kolommen, witruimtes en marges de volgende breedtes in pixels.
+### Breedtes per breakpoint
+
+Op de drie breakpoints hebben de kolommen, witruimtes en marges de volgende breedtes in pixels.
 
 #### Ruimtelijk
 
-| Naam variant | Vanaf vensterbreedte | Aantal kolommen | Breedte kolom | Breedte witruimte | Breedte marge | Breedte grid |
-| :----------- | -------------------: | --------------: | ------------: | ----------------: | ------------: | -----------: |
-| smal         |                  320 |               4 |            56 |                16 |            24 |          272 |
-| middelbreed  |                  576 |               8 |            42 |                24 |            36 |          504 |
-| breed        |                 1088 |              12 |            44 |                40 |            60 |          968 |
+| Breakpoint  | Vanaf vensterbreedte | Aantal kolommen | Breedte kolom | Breedte witruimte | Breedte marge | Breedte grid |
```

Comment:
Is "breedte marge" the same as gutter?

Reply:
No, gutters are between the columns and margins at the sides of the grid. ‘Witruimte’ is gutter.